modules/tinymce/CHANGELOG.md

```diff
@@ -70,1425 +70,1425 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Fixed directionality of UI when using an RTL language. #AP-423
+- Fixed page responsiveness with multiple inline editors. #AP-430
+- Fixed empty toolbar groups appearing through invalid configuration of the `toolbar` property. #AP-450
+- Fixed text not being retained when updating links through the link dialog. #AP-293
+- Fixed edit image context menu, context toolbar and toolbar items being incorrectly enabled when selecting invalid images. #AP-323
+- Fixed emoji type ahead being shown when typing URLs. #AP-366
+- Fixed toolbar configuration properties incorrectly expecting string arrays instead of strings. #AP-342
+- Fixed the block formatting toolbar item not showing a "Formatting" title when there is no selection. #AP-321
+- Fixed clicking disabled toolbar buttons hiding the toolbar in inline mode. #AP-380
+- Fixed `EditorResize` event not being fired upon editor resize. #AP-327
+- Fixed tables losing styles when updating through the dialog. #AP-368
+- Fixed context toolbar positioning to be more consistent near the edges of the editor. #AP-318
+- Fixed table of contents plugin now works with v5 toolbar APIs correctly. #AP-347
+- Fixed the `link_context_toolbar` configuration not disabling the context toolbar. #AP-458
+- Fixed the link context toolbar showing incorrect relative links. #AP-435
+- Fixed the alignment of the icon in alert banner dialog components. #TINY-2220
+- Fixed the visual blocks and visual char menu options not displaying their toggled state. #TINY-2238
+- Fixed the editor not displaying as fullscreen when toggled. #TINY-2237
 
 ### Removed
-- Removed the tox-custom-editor class that was added to the wrapping element of codemirror #TINY-2211
+- Removed the tox-custom-editor class that was added to the wrapping element of codemirror. #TINY-2211
 
 ## 5.0.0-preview-3 - 2018-10-18
 
 ### Changed
-- Changed editor layout to use modern CSS properties over manually calculating dimensions #AP-324
-- Changed `autoresize_min_height` and `autoresize_max_height` configurations to `min_height` and `max_height` #AP-324
-- Changed `Whole word` label in Search and Replace dialog to `Find whole words only` #AP-387
+- Changed editor layout to use modern CSS properties over manually calculating dimensions. #AP-324
+- Changed `autoresize_min_height` and `autoresize_max_height` configurations to `min_height` and `max_height`. #AP-324
+- Changed `Whole word` label in Search and Replace dialog to `Find whole words only`. #AP-387
 
 ### Fixed
-- Fixed bugs with editor width jumping when resizing and the iframe not resizing to smaller than 150px in height #AP-324
-- Fixed mobile theme bug that prevented the editor from loading #AP-404
+- Fixed bugs with editor width jumping when resizing and the iframe not resizing to smaller than 150px in height. #AP-324
+- Fixed mobile theme bug that prevented the editor from loading. #AP-404
 - Fixed long toolbar groups extending outside of the editor instead of wrapping
```

Comment:
Not sure if we are worried about these but just noting it
```suggestion
- Fixed long toolbar groups extending outside of the editor instead of wrapping.
```